app/js/analyzer.js

```diff
@@ -168,6 +206,20 @@ class Analyzer {
       const props = pick(searchParams, ["t", "ec", "ea", "cd35", "el"]);
       const summary = `${this.renderType(type)} ${props.join(":")}`;
       this.newMessage({type, summary, status, searchParamsList});
+    } else if (url.match(/\/\/tracking\.[a-z]+\.se\//)) {
+      // Bonnier News Analytics
+      if (request.method !== "POST") return;
+
+      const type = "BNA";
+      const postData = getJsonPostData(request);
+
+      if (!postData) return;
+
+      postData.forEach((data) => {
+        const props = ["event_name"].map((key) => data[key]).filter(Boolean);
```

Comment:
Räcker inte:
```suggestion
        const props = data["event_name"].filter(Boolean);
```
Eller ska det komma med fler? Enklare läsa ovan om det bara en den proppen det gäller

Reply:
Detta är hur funktionen [`pick`](https://github.com/BonnierNews/linda/blob/master/app/js/analyzer.js#L4-L6) tar ut data. Tanken är att det ska vara flera nycklar i array:en. Jag skrev om den först, men lintern tillåter inte optional chaining, så jag inline:ade funktionen istället.

Förslaget kommer inte fungera, då `data["event_name"]` är en sträng. Jag kan skriva om `pick` att hantera båda fallen, så blir det enhetligt.